libs/permission-checker/src/Check/RunnerSyncApi/CanResolveSharedCodeAndVariables.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\PermissionChecker\Check\RunnerSyncApi;
+
+use Keboola\PermissionChecker\Exception\PermissionDeniedException;
+use Keboola\PermissionChecker\PermissionCheckInterface;
+use Keboola\PermissionChecker\StorageApiToken;
+
+class CanResolveSharedCodeAndVariables implements PermissionCheckInterface
```

Comment:
Tady bych klidne vypustil ten `SharedCode` a pojmenoval to proste `CanResolveConfigVariables` (podle te akce). Knihovna se jmenuje `configuration-variables-resolver`, akce se jmenuje `ResolveConfigVariablesAction` apod. To ze to resolvuje i share codes mi prislo jako jedna z internich features. Stejne jako to ted bude resolvovat i hodnoty z Vaultu.

Reply:
Přejmenoval jsem to